.changeset/sharp-frogs-sing.md

```diff
@@ -2,6 +2,9 @@
 "@rhds/elements": patch
 ---
 
-* Fixes missing font-family stacks when base css isn't applied
-* Adds rh-token CSS custom properties for font family stacks
-* Fixes regression in spacing for slotted cta
+css:
+  - added missing `font-family` stacks when base css isn't applied
+  - [Red Hat Design Tokens][tokens] for font family stacks
```

Comment:
Should this be a link?

Reply:
I see, it's a reference to the link below.  Is that because links look bad in raw Markdown?

Reply:
Six of one...

I like the shorter lines